ParsingInputFile/parser.cpp

```diff
@@ -38,7 +37,9 @@ vector<string> readInputFile(const string& filepath) {
 vector<string> getRegularExpressions(const vector<string>& lines) {
     vector<string> regularExpressions;
     for (const auto& line : lines) {
-        if (line.find(':') != string::npos) {
+        if (   line.find(':') != string::npos
```

Comment:
Let's consider we have a rule like the following one:
RANGE = \(NUMBER \: NUMBER\)
Although it is a regular definition, it will be incorrectly considered as a regular expression.